app/src/main/java/com/glowstudio/android/blindsjn/feature/paymanagement/PayManagementScreen.kt

```
@@ -239,113 +383,159 @@ fun PayManagementScreen(
+                val dailyComparison by viewModel.dailyComparison.collectAsState()
+                val todaySales = salesSummary?.summary?.totalSales ?: 0.0
+                
                 Card(Modifier.fillMaxWidth().padding(bottom = 16.dp), shape = RoundedCornerShape(20.dp), colors = CardDefaults.cardColors(containerColor = CardWhite)) {
                     Column(Modifier.padding(16.dp)) {
-                        Text("금요일 평균 매출", fontWeight = FontWeight.Bold, fontSize = 18.sp)
+                        Text("오늘의 매출", fontWeight = FontWeight.Bold, fontSize = 18.sp)
                         Spacer(Modifier.height(8.dp))
-                        Text("₩ 450,000", fontSize = 24.sp, fontWeight = FontWeight.Bold, color = Blue)
-                        Text("다른 요일 대비 +25%", color = Color(0xFF388E3C), fontSize = 14.sp)
+                        Text(
+                            "₩ ${String.format("%,d", todaySales.toInt())}",
```

detekt (1.23.8)

[warning] String.format("%,d", todaySales.toInt()) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)